python/cudnn/gemm/frost/kernel_templates/sm100_matmul_mainloop_1ctamma.py

```
@@ -416,7 +422,34 @@ def _kernel(
                 if elect_one:
                     nvvm.mbarrier_arrive_expect_tx(b_full_mbar_ptr.subview(stage), sB_tma_bytes)
 
-                if cutlass.const_expr(multicast_a):
+                if cutlass.const_expr(a_mcast_slices > 1):
```

Ruff (0.16.1)

[error] Undefined name `a_mcast_slices`

(F821)